serialization-jackson/src/main/kotlin/io/datalbry/precise/serialization/jackson/exception/InvalidEnumValueException.kt

```diff
@@ -0,0 +1,6 @@
+package io.datalbry.precise.serialization.jackson.exception
+
+import io.datalbry.precise.api.schema.type.EnumType
+
+class InvalidEnumValueException(type: EnumType, value: String) :
+    IllegalArgumentException("Value \"${value}\" is not contained by type \"${type.name}\" with values: ${type.values}")
```

Comment:
We are logging values using the following format `[<message>]`

Reply:
"Value \"${value}\" is not contained by type \"${type.name}\" with values: ${type.values}"

should be the following:

"Value[${value}] is not contained by EnumType[${type.name}] with values ${type.values}"

Reply:
+ missing new line